runtime/modules/java/core/src/main/java/io/atlasmap/java/inspect/ClassInspectionService.java

```diff
@@ -66,7 +63,6 @@
             "java.util.concurrent.ConcurrentHashMap", "java.util.concurrent.ConcurrentMap"));
     private AtlasConversionService atlasConversionService = null;
     private List<String> fieldBlacklist = new ArrayList<String>(Arrays.asList("serialVersionUID"));
-    private List<String> classNameBlacklist = new ArrayList<String>();
```

Comment:
I thought we agreed several times with @mattrpav that the class name black list is a place holder for the future use and we shouldn't remove it.